bitcoin-core-dev-tech/2018-03-05-cross-curve-atomic-swaps.es.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Intercambios atómicos de curvas transversales (2018-03-05)
+transcript_by: Bryan Bishop
+translation_by: Blue Moon
+categories: ['core-dev-tech']
+tags: ['ux', 'lightning', 'schnorr']
```

Comment:
Probably because those are the tags in the original transcript. Reading that, it seems that indeed those tags are not relevant, so @BlueeeMoon if you change them here you can also change them in the original.